api.ts

```diff
@@ -1737,7 +1762,7 @@ export type MetadataContext = Record<string, any> & {__brand: 'MetadataContext'}
 /**
  * The type of values that can be returned from a {@link MetadataFormula}.
  */
-export type MetadataFormulaResultType = string | number | MetadataFormulaObjectResultType;
+export type MetadataFormulaResultType = string | number | MetadataFormulaObjectResultType | boolean;
```

Comment:
So my fear that I mentioned last week is that merely adding to the return type here makes metadata formula typing worse across the board. It's not the case that other metadata formulas can return booleans, only this one can.

We have this problem already, for example getConnectionName should only be able to return a string but it's typed as being able to return arrays and objects.

So I was hoping we would stem the bleeding here and allow for return type parameterization, so that this new metadata formula is typed only as returning a boolean and we don't introduce boolean as a new return type for existing metadata formulas.

If it's too gross, I suppose we could introduce boolean here as-is and do the parameterization cleanup separately.

Reply:
@gary-codaio Is this sufficient to allow declaring the new formula with a parameterized type of only `boolean` and have it behave properly? https://github.com/coda/packs-sdk/pull/new/jlg-metadata-parameterizatin

Reply:
I think that works for me to use. Want to merge or should I just add that in this change?

Reply:
Would just add it to your change.